### PR TITLE
Vectra fetch incident sort fix

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/38684

## Description
Added a fix for the fetch incidents command - Added the `order_field` to the fetch params so the incidents will be sorted by `last_timestamp` field. 

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by barchen1** [comment]
@guykeller SDK-validate failed cause `The integration Vectra v2 is currently in skipped`.

File: Packs/Vectra/Integrations/Vectra_v2/Vectra_v2.py [hunk 1]
```diff
@@ -141,7 +141,9 @@ def fetch_incidents(self, last_run: Dict):
             'query_string': query_string,
             'page_size': self.fetch_size,
             'page': 1,
+            'order_field': 'last_timestamp'
         }
+        demisto.debug(f'\n\nparams: {params}\n\n')
```
Comment (guykeller) [inline Packs/Vectra/Integrations/Vectra_v2/Vectra_v2.py:146]: If we want to leave this here let's remove the newline chars and write a better message

File: Packs/Vectra/Integrations/Vectra_v2/Vectra_v2.py [hunk 2]
```diff
@@ -582,6 +584,7 @@ def module_test(client: Client, last_run: dict):
 
 
 # COMMANDS MANAGER / SWITCH PANEL #
+
```
Comment (guykeller) [inline Packs/Vectra/Integrations/Vectra_v2/Vectra_v2.py:587]: intentional?

**Review by guykeller — CHANGES_REQUESTED** [review]

**Review by guykeller — APPROVED** [review]

**Comment by guykeller** [comment]
@yuvalbenshalom Can we please force merge? I'm accountable.
The integration needs to stay skipped.

**Pull request merged**